app/grandchallenge/challenges/emails.py

```diff
@@ -1,18 +1,25 @@
+from django.conf import settings
+from django.contrib.auth import get_user_model
 from django.contrib.sites.models import Site
-from django.core.mail import mail_managers
+from django.core.mail import mail_managers, send_mail
+from django.db.models import Q
+from django.utils.html import format_html
 
 from grandchallenge.subdomains.utils import reverse
 
 
 def send_challenge_created_email(challenge):
     site = Site.objects.get_current()
-    message = (
-        f"Dear manager,\n\n"
-        f"User {challenge.creator} has just created the challenge "
-        f"{challenge.short_name} at {challenge.get_absolute_url()}.\n\n"
-        f"Regards,\n"
-        f"{site.name}\n\n"
-        f"This is an automated service email from {site.domain}."
+    message = format_html(
+        "Dear manager,\n\n"
+        "User {} has just created the challenge {} at {}.\n\n"
+        "Regards,\n{}\n\n"
+        "This is an automated service email from {}.",
+        challenge.creator,
+        challenge.short_name,
+        challenge.get_absolute_url(),
+        site.name,
+        site.domain,
```

Comment:
Suggestion for the future - when you have a lot of arguments to `format_html` it is nicer to use kwargs as tracking these becomes difficult, eg

```
format_html(
        "Dear manager,\n\n"
        "User {user} has just created the challenge {challenge_name} at {challenge_url}.\n\n"
        "Regards,\n{site_name}\n\n"
        "This is an automated service email from {site_domain}.",
        user=challenge.creator,
        challenge_name=challenge.short_name,
        challenge_url=challenge.get_absolute_url(),
        site_name=site.name,
        site_domain=site.domain,
```